rpc/grpc/server/services/blockservice/service.go

```diff
@@ -0,0 +1,130 @@
+package blockservice
+
+import (
+	context "context"
+	"fmt"
+
+	"github.com/cometbft/cometbft/libs/log"
+	cmtpubsub "github.com/cometbft/cometbft/libs/pubsub"
+	blocksvc "github.com/cometbft/cometbft/proto/tendermint/services/block/v1"
+	"github.com/cometbft/cometbft/store"
+	"github.com/cometbft/cometbft/types"
+	"github.com/google/uuid"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type blockServiceServer struct {
+	store    *store.BlockStore
+	eventBus *types.EventBus
+	logger   log.Logger
+}
+
+// New creates a new CometBFT version service server.
+func New(store *store.BlockStore, eventBus *types.EventBus, logger log.Logger) blocksvc.BlockServiceServer {
+	log := logger.With("module", "grpc-block-service")
```

Comment:
The "module" field is already taken by the server: https://github.com/cometbft/cometbft/blob/902631a5f58acf0888c2ae0a98a63ff55f4a101f/rpc/grpc/server/server.go#L60

Rather:

```suggestion
	log := logger.With("service", "BlockService")
```